packages/gatsby-recipes/src/joi-to-graphql/index.js

```diff
@@ -1,2 +1,9 @@
-exports.transmuteType = exports.type = require(`./methods/compose-type`)
-exports.transmuteSchema = exports.schema = require(`./methods/compose-schema`)
+import transmuteType from "./methods/compose-type"
+import transmuteSchema from "./methods/compose-schema"
+
+export {
+  transmuteType,
+  transmuteType as type,
+  transmuteSchema,
+  transmuteSchema as schema,
+}
```

Comment:
For optimal tree-shaking 🙄 

```suggestion
export { default as transmuteType }  from "./methods/compose-type"
export { default as type }  from "./methods/compose-type"
export { default as transmuteSchema } from "./methods/compose-schema"
export { default as schema} from "./methods/compose-schema"
```